website/docs/cloud-docs/api-docs/run.mdx

```diff
@@ -53,55 +53,56 @@ Alternatively, you can create a run with a pre-existing configuration version, e
+| `policy_checked`       | The sentinel policy checking phase of a run has completed.                                                                                                                                                                                                                                                                                                                                                                                  |
+| `confirmed`            | The plan produced by the run has been confirmed.                                                                                                                                                                                                                                                                                                                                                                                            |
+| `post_plan_running`    | The post-plan phase of the run is in progress.                                                                                                                                                                                                                                                                                                                                                                                              |
+| `post_plan_completed`  | The post-plan phase of the run has completed.                                                                                                                                                                                                                                                                                                                                                                                               |
+| `planned_and_finished` | The completion of a run containing a plan only, or a run that produces a plan with no changes to apply. This is a final state.                                                                                                                                                                                                                                                                                                              |
+| `apply_queued`         | Once the changes in the plan have been confirmed, the run run will transition to `apply_queued`. This status indicates that the run should start as soon as the backend services that run terraform have available capacity. In Terraform Cloud, you should seldom see this status, as our aim is to always have capacity. However, in Terraform Enterprise this status will be more common due to the self-hosted nature.                  |
```

Comment:
Little typo here: `... been confirmed, the **run run** will transition ...`

Reply:
Fixed.  Thanks